src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -48,39 +50,63 @@ public class FindCommand extends Command {
 
     public static final String MESSAGE_SCORE_GREATER_THAN_MAX = "Value cannot be greater than the maximum score.";
 
-    private final PersonDetailContainsKeywordPredicate predicate;
+    private static final Logger logger = LogsCenter.getLogger(FindCommand.class);
+
+    private final Prefix prefix;
+    private final String keyword;
 
-    public FindCommand(PersonDetailContainsKeywordPredicate predicate) {
-        this.predicate = predicate;
+    /**
+     * Creates a FindCommand to find the specified Persons
+     */
+    public FindCommand(Prefix prefix, String keyword) {
+        this.prefix = prefix;
+        this.keyword = keyword;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        Exam selectedExam = model.getSelectedExam().getValue();
-        if (predicate.isExamRequired()) {
-            updateFilteredPersonListByScore(model, selectedExam);
+        logger.info("Executing find command");
```

Comment:
Great use of logging!

Reply:
Thanks! I will continue to incorporate logging throughout my code in the coming PRs